src/drone_controller/input_layer/drone_state.py

```diff
@@ -0,0 +1,9 @@
+# pylint: disable=too-few-public-methods
+# keep this only until this is implemented
+
+
+class DroneState:
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205179)